### ISSUE TITLE
RTL8723BE single antenna, Bluetooth and Wifi cannot coexist for audio transmission

### ISSUE THREAD

**Issue by german-rios-gonzalez** [lwfinger/rtlwifi_new#120]
tl;dr:
Wi-Fi On + Bluetooth On = Bluetooth cannot transmit audio.
Wi-Fi Off + Bluetooth On = Bluetooth can transmit audio.
I changed the antenna in the two positions and the result is the same.

Long story:

Hi mate, I have a brand new laptop from HP, the model is HP-15-an001la, comes with wifi-bluetooth combo from realtek, and HP only put only one antenna on there, I unboxed the laptop and I verified that the hadware in Windows was ok, Wi-Fi, Bluetooth, Display, etc. everything works so I erased the hard drive and I installed Fedora 23, the Wi-Fi does not work if I do not sit right next to the router. 

I searched for a solution, and found that for the range of the Wi-Fi the problem can be solve changing the antenna position, so I void the warranty and opened, changed the antenna position and the Wi-Fi range was ok, the connection of the Wi-Fi was droped after some minutes, so I do a further research and found the parameter fwlps set to 0 in this module do the trick, the Wi-Fi connection was going ok, I go outside out of the range of my Wi-Fi connection and bring my LG HBS750 bluetooth audio head set, it works fine, later I come back to the range of my Wi-Fi, my head set says: "Out of range, trying to reconnect", and I figure it out that the problem was the Wi-Fi connection.

If I turn off the Wi-Fi the bluetooth has no problems with the audio, but if the Wi-Fi is On and connected the bluetooth cannot transmit audio, I searched again and found that some wifi-bluetooth combos needs to be attached to a specific antenna position, so I set the parameter ant_sel=1 and changed the position of the antenna again, the Wi-Fi range is ok but the problem with the bluetooth audio transmit remains, I'm out of ideas that is because I'm reporting this as an issue in this module.

I'm a mechanical engineer so I have a very few skills to go further in this matter, but I can follow your instructions for give you more details about my machine or if you can tell me where I need to go to report this kind of problem I will do it.

P.S. I'm sorry for my english, is not my mother language.

**Comment by soyzenitram** [comment]
HP Pavillion also.

I am in the same situation: Wi-fi on, not any bluetooth communication; Wi-fi off, BT ok. Not only sound, also BT discovery and every BT activity is down when wifi is on.

Thanks a lot.

**Comment by ariancruz** [comment]
I'm in the similar situation, but in the opposite direction BT works fine and Wi-fi does not see cnx more than 2m lag in the games i'm using w10 any solution

**Comment by lwfinger** [comment]
A completely new version of the drivers has just been pushed. It fixes a lot of things. As a result, I am closing all open issues. If you get a new one, please create a new issue.

Note: The new driver fixes the problems with power save. It is recommended that you clear all ips, swlps, and fwlps options.
